src/hotspot/share/gc/shenandoah/shenandoahEvacOOMHandler.inline.hpp

```diff
@@ -32,18 +32,17 @@
 #include "runtime/atomic.hpp"
 
 void ShenandoahEvacOOMHandler::enter_evacuation(Thread* thr) {
-  jint threads_in_evac = Atomic::load_acquire(&_threads_in_evac);
```

Comment:
This load seems to be redundant. I don't think it has any ordering effects and we will load it again immediately either below or in `register_thread()`.